src/consts.ts

```diff
@@ -11,4 +11,6 @@ export const DEFAULT_TEXTURE_RESOLUTION = 10
 export const DEFAULT_EDGE_POINTS = 20
 export const DEFAULT_EDGE_MINIMUM_BEZIER = 64
 export const DEFAULT_HORIZONTAL_SCALE_MULTIPLIER = 0.25
-export const DEFAULT_HORIZONTAL_SCALE = 1
\ No newline at end of file
+export const DEFAULT_HORIZONTAL_SCALE = 1
+export const DEFAULT_GUIDES_COUNT = 30
```

Comment:
I'm thinking I may not need 30.. unless the user has a large screen. Might be worth calculating an amount instead of using a fixed one.

Reply:
Could also just create them on the fly when I need them so there'll only ever be as many as neccesary..

Reply:
Might experiment with this idea separately but will leave it for now.